catalog.html

```diff
@@ -0,0 +1,235 @@
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="service3">
+                <input id="service3" type="checkbox" name="Wi-Fi">Wi-Fi
+              </label>
+            </li>
+          </ul>
+        </div>
+        <div class="choice2">
+          <h3 class="catalog-filter-title">Тип жилья:</h3>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label for="place1">
+                <input id="place1" type="radio" name="hotel" value="shaving-supplies" checked>Гостиница
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place2">
+                <input id="place2" type="radio" name="motel" value="shaving-supplies">Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place3">
+                <input id="place3" type="radio" name="apartments" value="shaving-supplies">Апартаменты
+              </label>
+            </li>
+          </ul>
+        </div>
+        <div class="choice3">
+          <h3 class="catalog-filter-title">Стоимость, ₽:</h3>
+          <label for="format1">
+            <input id="format1" type="text" placeholder="от">
+          </label>
+          <label for="format2">
+            <input id="format2" type="text" placeholder="до">
+          </label>
+
+          <label for="roller">
+            <input type="range" id="roller" name="cost" min="0" max="9000" step="1000">
+          </label>
+        </div>
+        <button class="button" type="submit">Применить</button>
+        <button class="visually-hidden" type="reset">Сбросить</button>
+      </form>
+    </section>
+    <section>
+       <form action="https://echo.htmlacademy.ru/" method="get"></form>
+       <h2>Найдено гостиниц:</h2><output name="number">38</output>
+      <select name="select-control" aria-label="Сортировка">
+        <option value="popular">Сначала популярное</option>
+        <option value="cheap" selected>Сначала дешёвые</option>
+        <option value="expensive">Сначала дорогие</option>
+      </select>
+        <a class="button" href="#"><img src="" alt="Показать карточки."></a>
+        <a href="#" class="button"><img src="" alt="Показать слайды."></a>
+        <a href="#" class="button button-frame"><img src="" alt="Показать список."></a>
+
+        <ul class="housing-list">
+          <li class="housing-cards">
+            <img src="" alt="Гостиница в скале.">
+            <a class="housing-cards-link" href="housing.html">
+              <h3 class="housing-cards-title">Amara Resort&Spa</h3>
+            </a>
+            <p>Гостиница</p><p>От 4000 ₽</p>
+            <a class="button" href="housing.html">Подробнее</a>
+            <button class="checkbox" type="button">В избранное</button>
+            <img src="" alt="4 stars"> <p>Рейтинг 8,5</p>
```

Comment:
```suggestion
            <img src="" alt="4 stars."> <p>Рейтинг 8,5</p>
```

Всегда надо ставить точки в альтах

Reply:
поправь дальше по коду все аналогичные ошибки